src/core/server.h

```diff
@@ -78,9 +78,12 @@ class InferenceServer {
   // ownership of 'request'.
   Status InferAsync(std::unique_ptr<InferenceRequest>& request);
 
-  // Update the ServerStatus object with the status of the model. If
-  // 'model_name' is empty, update with the status of all models.
-  Status GetStatus(ServerStatus* server_status, const std::string& model_name);
+  // Return 'ready_model_versions' for 'model_name' and 'model_versions'.
+  // If 'model_version' is -1, update with all ready versions of 'model_name'.
+  // If 'model_name' is empty, update with the all ready versions of all models.
+  Status GetReadyModelVersions(
```

Comment:
There is already ModelIsReady() and ModelReadyVersions()

Reply:
These functions query on a particular model, we can utilize them but we still need to obtain a list of serving models first, and in the process we already have the ready information (from `GetLiveBackends()`). Or should I just update `ModelReadyVersions()` to be able to query on all models?